eventlog/el/main_test.go

```diff
@@ -21,7 +21,7 @@ const blockInterval = 1 * time.Second
 
 func TestCli(t *testing.T) {
 	// TODO: Fix this test.
-	t.Skip("Disabled for now.")
+	t.Skip("Disabled for now; we would need to test ol and el at the same time?")
```

Comment:
The Java tests do it by starting the service via Docker, then talking to it. I would like to maybe look into something like that for Go-based integration tests.

I would like very much to *not* write more shell code, I find shell scripts longer than 10 lines to be better in a typesafe language with a nice big library to use.

Reply:
More discussion needed, as I think that shell scripts give:
- _real_ integration testing, where we use _all_ layers
- insight into how users will use the script - if something is awkward to do in `test.sh`, the users will also have their problems
- can integrate multiple 'commands' if needed

Reply:
I do agree that testing using shell scripts have their use cases, but our home-brew framework isn't exactly easy to understand and use, I also struggle with it. In my previous job, we used the testing framework from FreeBSD - https://www.freebsd.org/cgi/man.cgi?query=atf-sh&sektion=3&apropos=0&manpath=FreeBSD+11.2-RELEASE+and+Ports, I'm sure there are many others. Perhaps we can think about using a shell testing framework, with nice documentation, for tests that must be done in shell scripts so that it's easier to learn/use and a bit more "industry standard".

Reply:
Sounds like a good idea. Let's have a look at those...